config/development.json

```diff
@@ -0,0 +1,9 @@
+{
+  "history": {
+    "author": {
+      "name": "AmbNum Bot Dev",
+      "email": "ambnum.bot.dev@disinfo.quaidorsay.fr"
```

Comment:
> I'm not sure I see the point of having a different email address in dev 🙂

Reply:
I agree, not useful at all! (dev's fantasy… :D)